client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx

```diff
@@ -187,7 +188,10 @@ const ProcessingStep: React.FC< ProcessingStepProps > = function ( props ) {
 		return <TailoredFlowPreCheckoutScreen flowName={ flowName } />;
 	}
 
-	if ( [ HUNDRED_YEAR_PLAN_FLOW, HUNDRED_YEAR_DOMAIN_FLOW ].includes( flowName ) ) {
+	if (
+		[ HUNDRED_YEAR_PLAN_FLOW, HUNDRED_YEAR_DOMAIN_FLOW ].includes( flowName ) ||
+		props.variantSlug === HUNDRED_YEAR_DOMAIN_TRANSFER
```

Comment:
I'm a little confused by the use of a variant for the 100-year transfer flow: can't we call the 100-year domain transfer flow `hundred-year-domain-transfer` and check for that flow name here? Or does the flow name have to actually be `domain-transfer` and we need to check the variant?

Reply:
Same as [this other comment](https://github.com/Automattic/wp-calypso/pull/95634#discussion_r1927473832)